coolwsd.xml.in

```diff
@@ -124,6 +124,8 @@
         <disable_server_audit type="bool" desc="Disabled server audit dialog and notification. Admin will no longer see warnings in the application user interface. This doesn't affect log file." default="false">false</disable_server_audit>
     </logging>
 
+    <canvas_slideshow_enabled type="bool" desc="If true, WebGl presentation rendered on the client side is enabled, otherwise interactive SVG is used." default="true">true</canvas_slideshow_enabled>
```

Comment:
can we move it at the end of file? now it is between logging options. at the end there are other on/off feature flags